Submitty/Submitty
# [Refactor:InstructorUI] Move Select2 Widget Code

### What is the current behavior?
Throughout Submitty, we repeat the code for creating and styling Select2 widgets.

### What is the new behavior?
Select2 widgets still look the same, but their code and styling has been to a few central files.

### How to test?
Go to a gradeable's edit page, select Submissions/Autograding, and see if you can still choose an autograding configuration successfully.

Go to Excused Absences Extensions on the sidebar, and see if you can still add an excused extension with a reason.

### Images
![image](https://github.com/Submitty/Submitty/assets/51209504/9790e163-0301-4170-a523-649377b5970c)

![image](https://github.com/Submitty/Submitty/assets/51209504/bbd2c78b-97c5-41cf-b82b-47bb0adf3f6d)

![image](https://github.com/Submitty/Submitty/assets/51209504/2cad2a23-eaea-45d9-a761-725fb90f7137)
